CHANGELOG.md

```diff
@@ -44,6 +48,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The Deep-copy feature of oeedger8r is now enabled by default.
 - The oeedger8r-generated header files now contain only the function prototypes. Marshalling structs, function id enums,
   and function tables are generated only in the c files.
+- Docs and scripts updated to use Azure DCAP client v1.6.0.
+- Fix #2930. Fixes the logic of detecting compilers when LVI mitigation is enabled. That is, the old logic always picks clang-7 (if installed) regardless of whether the environment variable CC is set to gcc.
+- Fix #2670. This fix also allows users to specify the version of clang (default is clang-7) when building the helloworld sample with LVI mitigation.
+- Fix #3056. oe_is_within_enclave() and oe_is_outside_enclave() now reflect the SGX enclave boundary as determined by the enclave SECS rather than the limit of the pages initially provisioned in to the enclave.
+- If not specified, CMAKE_BUILD_TYPE is set to Debug. This ensures that cmake and cmake -DCMAKE_BUILD_TYPE=Debug result in the same build configuration.
 
```

Comment:
Please add:

- Moved include/openenclave/attestation/plugin.h to internal. Currently only support internal attestation plugin registration.
- 'flags' parameter is removed from experimental function oe_get_evidence(). Use 'evidence_format' parameter to select evidence format.